apps/marketplace/components/BuyerTraining/BuyerTrainingResponseFormatsStage.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Form } from 'react-redux-form'
+import CheckboxDetailsField from 'shared/form/CheckboxDetailsField'
+import formProps from 'shared/form/formPropsSelector'
+import AUheadings from '@gov.au/headings/lib/js/react.js'
+import ErrorAlert from 'marketplace/components/Alerts/ErrorAlert'
+import styles from './BuyerTrainingResponseFormatsStage.scss'
+
+const atleastOneFormat = formValues =>
+  formValues.evaluationType &&
+  (formValues.evaluationType.includes('Written proposal') || formValues.evaluationType.includes('Response template'))
+
+const atleastOneProposal = formValues =>
+  !formValues.evaluationType.includes('Written proposal') ||
+  (formValues.evaluationType.includes('Written proposal') &&
+    formValues.proposalType &&
+    formValues.proposalType.length > 0)
+
+export const done = v => atleastOneFormat(v) && atleastOneProposal(v)
+
+const BuyerTrainingResponseFormatsStage = props => (
+  <Form
+    model={props.model}
+    validators={{
+      '': {
+        atleastOneFormat,
+        atleastOneProposal
+      }
+    }}
+    onSubmit={props.onSubmit}
+    onSubmitFailed={props.onSubmitFailed}
+    validateOn="submit"
+  >
+    <AUheadings level="1" size="xl">
+      Response formats
+    </AUheadings>
+    <ErrorAlert
+      title="An error occurred"
+      model={props.model}
+      messages={{
+        atleastOneFormat: 'You must choose what you would like sellers to provide through the Marketplace',
+        atleastOneProposal: 'You must select at least one proposal type.'
+      }}
+    />
+    <AUheadings level="2" size="md">
+      Select what sellers need to provide through the Marketplace:
+    </AUheadings>
+    <div className={styles.formats}>
+      <CheckboxDetailsField
+        model={`${props.model}.evaluationType[]`}
+        id={`response_format_proposal`}
+        name={`response_format_proposal`}
```

Comment:
Do these need to be wrapped in `{}`?  Applies elsewhere in this file too.

Reply:
updated